x/forwarding/keeper/keeper.go

```diff
@@ -59,8 +61,10 @@ func NewKeeper(
 
 		PendingForwards: collections.NewMap(transientBuilder, types.PendingForwardsPrefix, "pending_forwards", collections.StringKey, codec.CollValue[types.ForwardingAccount](cdc)),
 
-		authKeeper: authKeeper,
-		bankKeeper: bankKeeper,
+		authKeeper:     authKeeper,
+		bankKeeper:     bankKeeper,
+		channelKeeper:  channelKeeper,
+		transferKeeper: transferKeeper,
 	}
 
 	schema, err := builder.Build()
```

Comment:
Because IBC (where the Channel and Transfer keepers are from) doesn't support dependency injection yet, we have to pass `nil` in if we are using dependency injection!